mlflow/java/client/pom.xml

```diff
@@ -5,13 +5,13 @@
   <parent>
     <groupId>org.mlflow</groupId>
     <artifactId>mlflow-parent</artifactId>
-    <version>0.8.2</version>
+    <version>0.8.3</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
 
   <artifactId>mlflow-client</artifactId>
   <packaging>jar</packaging>
-  <version>0.8.2</version>
+  <version>0.8.3</version>
```

Comment:
By default, can we inherit this prop from `java/pom.xml` instead of redeclaring it here? In future, scoring/tracking my have different sub/version and different release schedules ... but for now, can we try and merge them if possible?